lib/burials/notification_email.rb

```diff
@@ -39,5 +41,10 @@ def personalization
 
       default.merge(burials)
     end
+
+    # @see VANotify::NotificationEmail::SavedClaim#callback_class
+    def callback_klass
```

Comment:
what is up with the spelling here?

Reply:
Does it matter that it is a bad idea?  :)

Reply:
:shrug: convention of Ruby (and other loose typed languages), although not entirely needed in this case